docs/configuration/session.md

```diff
@@ -36,11 +36,37 @@ session:
   # Note: the login portal must also be a subdomain of that domain.
   domain: example.com
 
+  # The remember me duration
+  # Value of 0 disables remember me
+  # Value is in seconds, or duration notation
+  remember_me_duration: "1"
```

Comment:
```suggestion
  remember_me_duration: "1m"
```

Reply:
Additionally move this above domain.

Reply:
Fixed

Reply:
Also doesn't need to be quoted so `1m` suffices.